gtfs/gtfs_utils/gtfs_utils/s3.py

```diff
@@ -90,3 +94,40 @@ def fetch_remote_file(remote_file_key: str,
     logging.debug(f'Finished file download (key="{remote_file_key}", local path="{local_file_full_path}")')
     return True
     # TODO: log file size
+
+
+def is_enough_disk_space(remote_file_keys: List[str],
+                         crud: S3Crud,
+                         max_download_size: float = None,
+                         local_dir: str = None,
+                         suppress_errors: bool = True) -> bool:
```

Comment:
Do we really need `suppress_errors`? If there's no enough space, maybe the code that calls `is_enough_disk_space` should throw an error, not this function itself.

Reply:
I thought about it, but than we would have to return all the relevant info to create an informative error message, or just have error message without the file sizes for example.

Reply:
As I understand, there are two cases:
1. When the remote files total size is larger than the configured max download size.
2. When the remote files total size is larger than the free space on the disk.

The current error message sounds like it's only the second case.

I think you should separate the checks, and throw separate errors.
I suggest changing the function(s) to return the amount of missing free space (in MB) instead of returning whether or not there's enough free space. This way, you can print the relevant information to the user without having to do anything apart from checking/comparing spaces.

Reply:
I moved the function to gtfs_stats so I'll be more comfortable using it for that scenario with those Errors, I still want to keep it one function so that you could easily validate your download size without using two functions.